cogs/utility.py

```diff
@@ -292,15 +292,13 @@ async def about(self, ctx):
         )
         embed.set_thumbnail(url=self.bot.user.avatar_url)
 
-        desc = "This is an open source Discord bot that serves as a means for "
-        desc += "members to easily communicate with server administrators in "
-        desc += "an organised manner."
+        desc = "This is an open source Discord bot that has been made by you."
         embed.description = desc
 
         embed.add_field(name="Uptime", value=self.bot.uptime)
         embed.add_field(name="Latency", value=f"{self.bot.latency * 1000:.2f} ms")
         embed.add_field(name="Version", value=f"`{self.bot.version}`")
-        embed.add_field(name="Authors", value="`kyb3r`, `Taki`, `fourjr`")
+        embed.add_field(name="Authors", value="`kyb3r`, `Taki`, `fourjr`, `TheSumit`,")
```

Comment:
The Authors List is for Core contributors who have made **significant** contributions to the bot, and are considered core developers. Redoing a single description, and removing the word pong, is not in any way, a significant contribution.